src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -577,10 +577,17 @@ namespace HVACVariableRefrigerantFlow {
                 OutdoorWetBulb = OutWetBulbTemp;
             }
         } else {
-            OutdoorDryBulb = OutDryBulbTemp;
-            OutdoorHumRat = OutHumRat;
-            OutdoorPressure = OutBaroPress;
-            OutdoorWetBulb = OutWetBulbTemp;
+            if (VRF(VRFCond).CondenserType == DataHVACGlobals::WaterCooled) {
+                OutdoorDryBulb = DataLoopNode::Node(VRF(VRFCond).CondenserNodeNum).Temp;
+                OutdoorHumRat = DataLoopNode::Node(VRF(VRFCond).CondenserNodeNum).HumRat;
```

Comment:
I do not know what value to set to. You are right, we should not set to any value. May be I should not have added the check for water cooled in the first place since the condenser inlet node number is zero. Because, that code section is executed only if the condenser node number is zero. I think, I have to revert this change.

Reply:
Reverted this part of the fix. No need to check for WaterCooled condenser type check if the condenser inlet node number is zero.